test.js

```diff
@@ -202,6 +203,13 @@ describe('chalk.level', () => {
 		assert.equal(chalk.level, 1);
 		chalk.level = oldLevel;
 	});
+
+	it('should disable colors if they are not supported', () => {
+		return execa('node', ['fixture'])
+			.then(result => {
+				assert.equal(result.stdout, 'test');
+			});
+	});
```

Comment:
Does this guarantee that the spawned `node` instance is not supported? I'm failing to see how this is any different than the tests now.